tmd/io/io.py

```diff
@@ -122,6 +122,9 @@ def load_population(neurons, tree_types=None, name=None):
 
     pop = Population.Population(name=name)
 
+    if any([i.endswith(".asc") for i in files]):
```

Comment:
How about sth like the following:

``` python

for filename in files:
        try:
            assert filename.endswith(('.h5', '.swc'))
            pop.append_neuron(load_neuron(i, tree_types=tree_types))
        except AssertionError:
                Warning("{} is not  a valid h5 or swc file".format(filename))
        except LoadNeuronError:
                print('File failed to load: {}'.format(i))
```

Reply:
Indeed! But more lines ;) I think the lines below only open .h5 and .swc files, so it will not try to open anything else. This was just a warning to say that there are some .asc files in the folder which are not red. I don't mind how to add this warning, I let you guys decide and merge if you want.

Reply:
Behaviour-wise the two versions are identical. If the file is not `.h5` or `.swc` then the assertion throws and the except is triggered before running the load_neuron line, producing a warning for the invalid extension.

Reply:
To be precise they are not exactly identical. My version will produce a warning with the filename name for each invalid morphology.

Reply:
I think we can go with this alternative.

Reply:
Done!